collectors/python.d.plugin/uwsgi/uwsgi.chart.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+# Description: uwsgi netdata python.d module
+# Author: Robbert Segeren (robbert-ef)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+from copy import deepcopy
+from bases.FrameworkServices.SocketService import SocketService
+
+# default module values (can be overridden per job in `config`)
+# update_every = 2
+priority = 60000
+retries = 60
+
+ORDER = ['requests', 'tx', 'avg_rt', 'memory_rss', 'memory_vsz', 'exceptions', 'harakiri', 'respawn', ]
+
+# NOTE: lines are created dynamically in `check()` method
+CHARTS = {
+    'requests': {
+        'options': [None, 'Requests', 'requests/s', 'requests', 'uwsgi.requests', 'stacked'],
+        'lines': [
+            ['requests.%s', 'worker %s', 'incremental']
+        ]},
```

Comment:
Please update these so that the formatting matches the other python.d modules.

In short, each chart should look like this:
```
'requests': {
    'options': [None, 'Requests', 'requests/s', 'requests', 'uwsgi.requests', 'stacked'],
    'lines': [
        ['requests.%s', 'worker %s', 'incremental']
    ]
},
```
With the closing braces for the 'lines' list and the chart itself each on their own line indented appropriately.

Reply:
done